### PR TITLE
chore: 

### PR DESCRIPTION
**Pull request by Abbie-n**

### CODE REVIEW

File: lib/ui/views/home/schedule_notifications/schedule_reminder/schedule_reminder_viewmodel.dart [hunk 1]
```diff
@@ -11,6 +13,9 @@ class ScheduleNotificationsViewModel extends BaseViewModel {
   int _selectedMonth;
   dynamic _selectedTime;
   DateTime _today = DateTime.now();
+  NotificationRemindersService reminders = NotificationRemindersService();
+  // This is temporary to give each notification a unique id
+  int random = Random().nextInt(100);
```
Comment (ufe-pr) [inline lib/ui/views/home/schedule_notifications/schedule_reminder/schedule_reminder_viewmodel.dart:18]: Use the uuid function instead to generate unique IDs. Check the store local data source for an example

File: lib/ui/views/home/signup/business/business_view.dart [hunk 2]
```diff
@@ -80,6 +80,8 @@ class _PartialBuildForm extends HookViewModelWidget<BusinessViewModel> {
                   Padding(
                     padding: EdgeInsets.all(SizeConfig.yMargin(context, 2)),
                     child: TextFormField(
+                      autofocus: true,
+                      textCapitalization: TextCapitalization.sentences,
```
Comment (ufe-pr) [inline lib/ui/views/home/signup/business/business_view.dart:84]: Names should use word capitalization

File: lib/ui/views/marketing/add_new_customer/add_new_customer_view.dart [hunk 3]
```diff
@@ -41,9 +46,11 @@ class AddNewCustomerView extends StatelessWidget {
                           padding: const EdgeInsets.all(8.0),
                           child: Container(
                             child: TextField(
+                              textCapitalization: TextCapitalization.sentences,
```
Comment (ufe-pr) [inline lib/ui/views/marketing/add_new_customer/add_new_customer_view.dart:49]: Names should use word caps

File: lib/ui/views/marketing/add_new_customer/add_new_customer_view.dart [hunk 4]
```diff
@@ -100,7 +107,8 @@ class AddNewCustomerView extends StatelessWidget {
                                 keyboardType: TextInputType.numberWithOptions(),
```
Comment (ufe-pr) [inline lib/ui/views/marketing/add_new_customer/add_new_customer_view.dart:107]: Do phone numbers need the options?

File: lib/ui/views/marketing/marketing_home_page/marketing_homepage_view.dart [hunk 5]
```diff
@@ -147,6 +147,7 @@ class MarketingHomePageView extends StatelessWidget {
                               horizontal: 15.0, vertical: 5),
                           child: Container(
                             child: TextField(
+                              textCapitalization: TextCapitalization.sentences,
```
Comment (ufe-pr) [inline lib/ui/views/marketing/marketing_home_page/marketing_homepage_view.dart:150]: Search fields don't need to be sentence capped

File: lib/ui/widgets/main/menu_buttons.dart [hunk 6]
```diff
@@ -35,9 +35,9 @@ class BusinessMenuOptions extends HookViewModelWidget<MainViewModel> {
         },
         child: Container(
           height: SizeConfig.yMargin(context, 12),
-          color: model.currStore.id == business.id
-              ? Color(0xFF2F80ED).withOpacity(0.2)
-              : Colors.transparent,
```
Comment (ufe-pr) [inline lib/ui/widgets/main/menu_buttons.dart:40]: Isn't there supposed to be an indicator to show the current store?

File: lib/ui/views/marketing/add_customer_message/add_customer_message_view.dart [hunk 7]
```diff
@@ -30,296 +30,304 @@ class AddCustomerMessageView extends StatelessWidget {
...
-                    hintText: AppLocalizations.of(context).typeCustomerName,
-                    prefixIcon: Icon(Icons.search),
-                    border: InputBorder.none,
-                    focusedBorder: InputBorder.none,
-                    enabledBorder: InputBorder.none,
-                    fillColor: Colors.transparent,
-                    focusColor: Colors.transparent,
+                  clipBehavior: Clip.hardEdge,
+                  child: TextField(
+                    textCapitalization: TextCapitalization.sentences,
```
Comment (ufe-pr) [inline lib/ui/views/marketing/add_customer_message/add_customer_message_view.dart:70]: Names should have word caps

File: lib/ui/views/home/signup/business/business_view.dart (hunk 2 above)
Comment (Abbie-n) [reply lib/ui/views/home/signup/business/business_view.dart:84]: Done that already.

File: lib/ui/views/home/schedule_notifications/schedule_reminder/schedule_reminder_viewmodel.dart (hunk 1 above)
Comment (Abbie-n) [reply lib/ui/views/home/schedule_notifications/schedule_reminder/schedule_reminder_viewmodel.dart:18]: Done

File: lib/ui/views/marketing/add_new_customer/add_new_customer_view.dart (hunk 3 above)
Comment (Abbie-n) [reply lib/ui/views/marketing/add_new_customer/add_new_customer_view.dart:49]: Done that already.

File: lib/ui/views/marketing/add_new_customer/add_new_customer_view.dart (hunk 4 above)
Comment (Abbie-n) [reply lib/ui/views/marketing/add_new_customer/add_new_customer_view.dart:107]: Done

File: lib/ui/views/marketing/marketing_home_page/marketing_homepage_view.dart (hunk 5 above)
Comment (Abbie-n) [reply lib/ui/views/marketing/marketing_home_page/marketing_homepage_view.dart:150]: It's a search field for names. I think it's appropriate

File: lib/ui/widgets/main/menu_buttons.dart (hunk 6 above)
Comment (Abbie-n) [reply lib/ui/widgets/main/menu_buttons.dart:40]: That was commented out.

File: lib/ui/views/marketing/add_customer_message/add_customer_message_view.dart (hunk 7 above)
Comment (Abbie-n) [reply lib/ui/views/marketing/add_customer_message/add_customer_message_view.dart:70]: I feel since they search for names, it is appropriate.

**Pull request merged**